### ISSUE TITLE
Increase button focus contrast (accessiblity)

### ISSUE THREAD

**Issue by NomeQ** [interline-io/calact-network-analysis-tool#354]
When triggering the keyboard focus state (:focus-visible) on the purple buttons throughout the project, Bulma's defaults provide insufficient contrast. 

<img width="323" height="113" alt="Image" src="https://github.com/user-attachments/assets/b0e134e2-7aaf-45f5-bf22-7339cd9edec1" />

Minimum contrast against the surrounding content should be 3:1. I'm not a huge fan of how difficult Bulma makes it to figure out what the exact color value is, since they are calculated and also use opacity, but it's obvious visually that the current outline does not have sufficient contrast against the button color.
